test/e2e/tests/tokens/token-list.spec.ts

```diff
@@ -42,6 +42,7 @@ describe('Token List', function () {
     await driver.clickElement(
       '[data-testid="import-tokens-modal-import-button"]',
     );
+    await driver.findElement({ text: 'Token imported', tag: 'h6' });
```

Comment:
wait until token modal is closed and token is properly imported before asserting